pkgs/by-name/ae/aerospace/package.nix

```
@@ -18,7 +18,7 @@ stdenv.mkDerivation {
 
   src = fetchzip {
     url = "https://github.com/nikitabobko/AeroSpace/releases/download/v${version}/AeroSpace-v${version}.zip";
-    sha256 = "sha256-IMU0s57dpes7Vm2Wv191LwkRgiF+ZIqNWHzrl4a1Pm0=";
+    sha256 = "sha256-46vSv/Hj36zF/WfZIBGBjGfC2683zFlYdlO9fc8BeHc=";
```

update hash to `0.18.2-Beta` version
```suggestion
    sha256 = "sha256-/Fc4Zk8KvAdaKXyHmeL9nh79CAQLx/Y6URFWIOL5YyQ=";
```